wallet/src/account/constants.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) const PARALLEL_REQUESTS_AMOUNT: usize = 500;
 
 /// ms before an account actually syncs with the network, before it just returns the previous syncing result
 /// this is done to prevent unnecessary simultaneous synchronizations
-pub(crate) const MIN_SYNC_INTERVAL: u128 = 200;
+pub(crate) const MIN_SYNC_INTERVAL: u128 = 5;
```

Comment:
5 Ms is basically nothing. Only literally sync(). Sync() would have an effect. Do we we want to keep this at all then?

Reply:
It can also happen if you call it multiple times in parallel and I think for that it makes sense to keep